include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -2764,15 +2768,15 @@ __pattern_fill(__parallel_tag<_IsVector>, _ExecutionPolicy&& __exec, _RandomAcce
     return __internal::__except_handler([&__exec, __first, __last, &__value]() {
         __par_backend::__parallel_for(__backend_tag{}, ::std::forward<_ExecutionPolicy>(__exec), __first, __last,
                                       [&__value](_RandomAccessIterator __begin, _RandomAccessIterator __end) {
-                                          __internal::__brick_fill<__parallel_tag<_IsVector>, _ExecutionPolicy, _Tp>{
-                                              __value}(__begin, __end, _IsVector{});
+                                          __internal::__brick_fill<__parallel_tag<_IsVector>, _Tp>{__value}(
+                                              __begin, __end, _IsVector{});
                                       });
         return __last;
     });
 }
 
-template <class _Tag, typename _ExecutionPolicy, typename _Tp>
-struct __brick_fill_n<_Tag, _ExecutionPolicy, _Tp, ::std::enable_if_t<__is_host_dispatch_tag_v<_Tag>>>
+template <class _Tag, typename _Tp>
+struct __brick_fill_n<_Tag, _Tp, std::enable_if_t<__is_host_dispatch_tag_v<_Tag>>>
```

Comment:
The same comment as above.
(So, we don't need std::enable_if_t here.)

Reply:
You are correct. But this point doesn't correlated with this PR goal, so I propose create separate issue.

Reply:
Issue created: https://github.com/uxlfoundation/oneDPL/issues/2179

Reply:
Fixed